api_schemas/event_signup_schemas.py

```diff
@@ -0,0 +1,11 @@
+from api_schemas.base_schema import BaseSchema
+
+
+class EventSignupCreate(BaseSchema):
+    priority: str | None
```

Comment:
If these are optional you also need `priority: str | None = None` so that `priority` defaults to `None` if requests does not specify priority.